test/commands/deploy/pipeline.test.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect, test } from '@oclif/test';
+
+describe('validate flags', () => {
```

Comment:
These tests are ok. I thought of adding new tests with the happy path but as the command is not fully implemented yet we may leave this for the future. What do you think?

Reply:
Test for happy path added, thx